Bacterial DNA extraction and 16S rRNA gene sequencing for human studies and mouse probiotics experiments: Sample preparation and sequencing was performed in collaboration with the Baylor College of Medicine Alkek Center for Metagenomics and Microbiome Research (BCM-CMMR) across the entire melanoma patient cohort (n=438, one sample per patient) and for all samples derived from the mouse probiotics experiment. The 16S V4 amplification protocol was adapted from those described in Earth Microbiome Project i.e., primers 515f/806r (45, 46). Briefly, genomic DNA was extracted using Qiagen/MO BIO PowerMicrobiome DNA isolation kit on automated liquid handlers (Hamilton STARlet instruments). The 16S rRNA gene V4 region was amplified by PCR and sequenced on the MiSeq platform (Illumina, Inc, San Diego, CA) using the 2x250 bp paired-end protocol yielding paired-end reads with near-complete overlap. The primers used for amplification contain adapters for MiSeq sequencing and single-end barcodes allowing pooling and direct sequencing of PCR products (46). 16S analytic pipeline: Raw FASTQ files were processed using DADA2 v1.18 to generate amplicon sequence variants (ASVs) by denoising 16S V4 amplicon data (47), with taxonomy assigned from SILVA v138 (48) using a naïve Bayesian classifier implemented by the assign Taxonomy function (https://www.rdocumentation.org/packages/dada2/versions/1.0.3/topics/assignTaxonomy) which implements the RDP classifier algorithm in Wang 2007 (49) with kmer size 8 and 100 bootstrap replicates. For the analysis, only ASVs that matched the following criteria were included: consensus sequences between 252-254 bp in length with a minimum read count of ≥10 reads across all samples, taxonomies defined to at least the family level, and excluding ASVs that mapped to Archaea or mitochondria. The filtered abundance table was rarefied to 3,319 reads per sample for alpha diversity calculations. All diversity calculations were done using a custom script (provided in Supplemental File) which performed the following steps: ASV richness was calculated by summing up the total number of ASVs per sample; Simpson Index scores were calculated using the following formula D= (∑n(n-1))/(N(N-1)) where n= read count for a particular ASV in a sample and N= total read count across all ASVs in that particular sample. These values were then inverted (1/D) to calculate the Inverse Simpson Score.![image](https://user-images.githubusercontent.com/7674705/151001540-7e9d60ee-81f7-458d-bf09-c0fb1ced9552.png)
